security/providers/header/src/main/java/io/helidon/security/providers/header/HeaderAtnProvider.java

```diff
@@ -312,6 +326,7 @@ public Builder optional(boolean optional) {
          * @param target outbound target
          * @return updated builder instance
          */
+        @ConfiguredOption(key = "outbound", description = "A list of outbound configurations.")
```

Comment:
If it is a list, it should be marked as one - this will be a single target only. Add kind = LIST